compiler/src/parsing/ast_helper.re

```diff
@@ -324,7 +324,7 @@ module Top = {
 };
 
 module Val = {
-  let mk = (~loc=?, ~mod_, ~name, ~alias, ~typ, ~prim) => {
+  let mk = (~loc=?, ~mod_, ~name, ~alias, ~typ, prim) => {
```

Comment:
I think we like having the labeled arguments everywhere. Would you mind changing this to
```suggestion
  let mk = (~loc=?, ~mod_, ~name, ~alias, ~typ, ~prim, ()) => {
```
in the places you removed the last labeled argument? (The call syntax is still the same as it was before without having to worry about that last unit.)

Reply:
(Actually not all of them, just this one.)

Reply:
Done 👍